Examples/DissipativeIsing1d/ising1d.py

```diff
@@ -15,7 +15,7 @@
 import netket as nk
 import numpy as np
 
-rg = nk.utils.RandomEngine(seed=1234)
+rg = nk.random.seed(seed=1234)
```

Comment:
`rg is None` here (and not used further down the line).
```suggestion
nk.random.seed(seed=1234)
```